spaceship/spec/tunes/device_type_spec.rb

```diff
@@ -1,5 +1,12 @@
 describe Spaceship::Tunes::DeviceType do
   describe "type identifiers" do
+    before(:each) do
+      # let's catch those calls to avoid polluting the output
+      # Note: warning has a different signature in newer versions of ruby
```

Comment:
Is this the reason why we have 2 `allow` calls? If so, can we specify this in the comment? I think it'd be more clear, like:

```suggestion
      # Note: warnings have different signature depending on the Ruby version, hence why we need more than one allow(…)
```

Reply:
I made an update.